src/components/Details/Details.js

```diff
@@ -35,21 +45,21 @@ function Details( { pokemon, setPokemonData } ) {
 
 			<div className="tabs-switch-container">
 
-				<button className={ getClassName( 'about' ) } data-tab="about" onClick={ switchTab }>About</button>
-				<button className={ getClassName( 'base-stats' ) } data-tab="base-stats" onClick={ switchTab }>Base Stats</button>
-				<button className={ getClassName( 'evolution' ) } data-tab="evolution" onClick={ switchTab }>Evolution</button>
+				<button className={ getClassName( TAB_ABOUT ) } data-tab={ TAB_ABOUT } onClick={ switchTab }>About</button>
+				<button className={ getClassName( TAB_STATS ) } data-tab={ TAB_STATS } onClick={ switchTab }>Base Stats</button>
+				<button className={ getClassName( TAB_EVOLUTION ) } data-tab={ TAB_EVOLUTION } onClick={ switchTab }>Evolution</button>
```

Comment:
Create an array and iterate.. [ { className: TAB_ABOUT, text: 'About' }, ... ]

Reply:
Done